docs/connect-field-devices/asset-implemetation.md

```diff
@@ -0,0 +1,57 @@
+# Asset implementation
+
+An **Asset** is a logical representation of a field device, described by a list of **Channels**. The Asset uses a specific Driver instance to communicate with the underlying device and it models a generic device resource as a **Channel**. A register in a PLC or a GATT Characteristic in a Bluetooth device are examples of Channels. In this way, each Asset has multiple Channels for reading and writing data from/to an Industrial Device.
+
+Assets can be used as Wire Components to access the resources referenced by the defined channels inside a Wire Graph, see the [Assets as Wire Components](doc:assets-as-a-wire-components) guide for more details. 
```

Comment:
Can you put `../kura-wires/assets-as-wire-components.md` as the link? So that it will point to the right section once PR https://github.com/eclipse/kura/pull/4165 gets merged.